pkg/types/baremetal/validation/platform_test.go

```diff
@@ -740,3 +272,169 @@ func TestValidatePlatform(t *testing.T) {
+	return hb
+}
+
+func (hb *hostBuilder) BMCPassword(value string) *hostBuilder {
+	hb.Host.BMC.Password = value
+	return hb
+}
+
+///////////////////////////////////////////////////////////////////////////////
+
+type platformBuilder struct {
+	baremetal.Platform
+}
+
+func platform() *platformBuilder {
+	return &platformBuilder{
+		baremetal.Platform{
+			APIVIP:                       "192.168.111.2",
+			DNSVIP:                       "192.168.111.3",
+			IngressVIP:                   "192.168.111.4",
+			Hosts:                        []*baremetal.Host{},
+			LibvirtURI:                   "qemu://system",
+			ProvisioningNetworkCIDR:      ipnet.MustParseCIDR("172.22.0.0/24"),
+			ClusterProvisioningIP:        "172.22.0.3",
+			BootstrapProvisioningIP:      "172.22.0.2",
+			ExternalBridge:               "br0",
+			ProvisioningBridge:           "br1",
+			ProvisioningNetworkInterface: "ens3",
+		}}
+}
+
+func (pb *platformBuilder) build() *baremetal.Platform {
+	return &pb.Platform
+}
+
+func (pb *platformBuilder) ProvisioningNetworkCIDR(value string) *platformBuilder {
+	pb.Platform.ProvisioningNetworkCIDR = ipnet.MustParseCIDR(value)
+	return pb
+}
+
+func (pb *platformBuilder) ClusterProvisioningIP(value string) *platformBuilder {
+	pb.Platform.ClusterProvisioningIP = value
+	return pb
+}
+
+func (pb *platformBuilder) BootstrapProvisioningIP(value string) *platformBuilder {
+	pb.Platform.BootstrapProvisioningIP = value
+	return pb
+}
+
+func (pb *platformBuilder) BootstrapOSImage(value string) *platformBuilder {
+	pb.Platform.BootstrapOSImage = value
+	return pb
+}
+
+func (pb *platformBuilder) ClusterOSImage(value string) *platformBuilder {
+	pb.Platform.ClusterOSImage = value
+	return pb
+}
+
+func (pb *platformBuilder) ProvisioningDHCPRange(value string) *platformBuilder {
+	pb.Platform.ProvisioningDHCPRange = value
+	return pb
+}
+
+func (pb *platformBuilder) APIVIP(value string) *platformBuilder {
+	pb.Platform.APIVIP = value
+	return pb
+}
+
+func (pb *platformBuilder) DNSVIP(value string) *platformBuilder {
+	pb.Platform.DNSVIP = value
+	return pb
+}
+
+func (pb *platformBuilder) IngressVIP(value string) *platformBuilder {
+	pb.Platform.IngressVIP = value
+	return pb
+}
+
+func (pb *platformBuilder) Hosts(value ...*baremetal.Host) *platformBuilder {
+	pb.Platform.Hosts = value
+	return pb
+}
+
+func (pb *platformBuilder) LibvirtURI(value string) *platformBuilder {
+	pb.Platform.LibvirtURI = value
+	return pb
+}
+
+func (pb *platformBuilder) ExternalBridge(value string) *platformBuilder {
+	pb.Platform.ExternalBridge = value
+	return pb
+}
+
+func (pb *platformBuilder) ProvisioningBridge(value string) *platformBuilder {
+	pb.Platform.ProvisioningBridge = value
+	return pb
+}
+
+func (pb *platformBuilder) ProvisioningNetworkInterface(value string) *platformBuilder {
+	pb.Platform.ProvisioningNetworkInterface = value
+	return pb
+}
+
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
again please drop all of these slashes. use multiline comments if you really want to explain the following code
https://golang.org/doc/effective_go.html#commentary